packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -52,6 +52,9 @@ This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aw
   - [Cancel Step](#cancel-step)
   - [Modify Instance Fleet](#modify-instance-fleet)
   - [Modify Instance Group](#modify-instance-group)
+- [EKS](#eks)
+  - [Create Nodegroup](#create-nodegroup)
```

Comment:
"Node group" should be two words: https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html

(Yes, it's a little confusing that the EKS API action is CreateNodegroup which makes it look like it should be one word.)